Kudu.TestHarness/TestRepositories.cs

```diff
@@ -9,59 +9,59 @@ internal class TestRepositories
+            new TestRepositoryInfo("https://github.com/KuduApps/MvcApplicationWithNuGetAutoRestore",    "1e269b5"),
+            new TestRepositoryInfo("https://github.com/KuduApps/MVCAppWithLatestNuget.git",             "7f49749"),
+            new TestRepositoryInfo("https://github.com/KuduApps/MvcAppWithTypeScript.git",              "7319dbb"),
+            new TestRepositoryInfo("https://github.com/KuduApps/NodeHelloWorldNoConfig.git",            "d8a15b6"),
+            new TestRepositoryInfo("https://github.com/KuduApps/NodeInnerSubDir.git",                   "58f93f8"),
+            new TestRepositoryInfo("https://github.com/KuduApps/NoDeployableProjects.git",              "04e9920"),
+            new TestRepositoryInfo("https://github.com/KuduApps/NodeWebHookTest.git",                   "cd5bee7"),
+            new TestRepositoryInfo("https://github.com/KuduApps/NodeWithCustomStartScript.git",         "a142e44"),
+            new TestRepositoryInfo("https://github.com/KuduApps/NodeWithSolutions.git",                 "bd16404"),
+            new TestRepositoryInfo("https://github.com/KuduApps/NpmSite.git",                           "2b29b9d"),
+            new TestRepositoryInfo("https://github.com/KuduApps/Orchard.git",                           "09ce5a5"),
+            new TestRepositoryInfo("https://github.com/KuduApps/PreviewMvc5.git",                       "f361ad3"),
+            new TestRepositoryInfo("https://github.com/KuduApps/PreviewSpa5.git",                       "9f6cde2"),
+            new TestRepositoryInfo("https://github.com/KuduApps/PreviewWebApi5.git",                    "74a0ce0"),
+            new TestRepositoryInfo("https://github.com/KuduApps/ProjectKWebApplication.git",            "fa5f2bc"), 
+            new TestRepositoryInfo("https://github.com/KuduApps/ProjectWithNoSolution.git",             "5460398"),
+            new TestRepositoryInfo("https://github.com/KuduApps/SpecificDeploymentConfiguration.git",   "fa99bb8"),
+            new TestRepositoryInfo("https://github.com/KuduApps/TargetPathTest.git",                    "7446104"),
+            new TestRepositoryInfo("https://github.com/KuduApps/VersionPinnedNodeJsApp.git",            "5f6c64f"),
+            new TestRepositoryInfo("https://github.com/KuduApps/VersionPinnedNodeJsAppCustom.git",      "8d43d4e"),
+            new TestRepositoryInfo("https://github.com/KuduApps/WaitForUserInput.git",                  "cb35610"),
+            new TestRepositoryInfo("https://github.com/KuduApps/WarningsAsErrors.git",                  "51b6a48"),
+            new TestRepositoryInfo("https://github.com/KuduApps/waws.git",                              "7652a66"),
+            new TestRepositoryInfo("https://github.com/KuduApps/WebDeploySamples.git",                  "ee3a42b"),
+            new TestRepositoryInfo("https://github.com/KuduApps/WebSiteInSolution.git",                 "efee8af"),
```

Comment:
I'd avoid changing all the spacing. It might give you hell merging into private, which has a different set of repos.